### PR TITLE
refactor(backend): remove admin create contributions

### LINKED ISSUE #2723: 🔧 [Refactor] Remove ` adminCreateContributions` in contribution resolver
## 🔧 Refactor ticket
As admin, it is no longer possible to create multiple creations, but the API call does still exist

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
remove admin create contributions

### Issues
- fixes #2723 

### Todo

- [X] None

### CODE REVIEW

**Review by ulfgebhardt — APPROVED** [review]

File: admin/src/components/CreationFormular.vue [hunk 1]
```diff
@@ -140,78 +135,38 @@ export default {
...
+          this.$emit('update-user-data', this.item, result.data.adminCreateContribution)
+          this.$store.commit('openCreationsPlus', 1)
+          this.toastSuccess(
+            this.$t('creation_form.toasted', {
+              value: this.value,
+              email: this.item.email,
+            }),
+          )
+          // what is this? Tests says that this.text is not reseted
+          this.$refs.creationForm.reset()
```
Comment (clauspeterhuebner) [inline admin/src/components/CreationFormular.vue:162]: Is this a ToDo or what should this comment express?

File: admin/src/components/NavBar.spec.js [hunk 2]
```diff
@@ -68,23 +68,20 @@ describe('NavBar', () => {
...
 
     afterEach(() => {
       delete window.location
       window.location = windowLocation
     })
 
-    it.skip('changes window location to wallet', () => {
-      expect(windowLocationMock()).toBe('valid-token')
+    it('changes window location to wallet', () => {
+      expect(window.location).toBe('http://localhost/authenticate?token=valid-token')
```
Comment (clauspeterhuebner) [inline admin/src/components/NavBar.spec.js:84]: Is this always an URL with localhost? What happens on other instances like stage1/2/3 or production?

**Review by clauspeterhuebner — COMMENTED** [review]
with some questions

**Review by Elweyn — CHANGES_REQUESTED** [review]
The changes made seem good but there are still references to massCreation in the admin interface. 
- admin/src/pages/UserSearch.vue
- admin/src/store/store.js
- admin/src/store/store.test.js

File: admin/src/components/CreationFormular.vue (hunk 1 above)
Comment (Mogge) [reply admin/src/components/CreationFormular.vue:162]: Out of scope of this PR. This should be addressed to @oderly

**Review by Elweyn — APPROVED** [review]

**Pull request merged**